xapian-applications/omega/docs/postprocessing.rst

```diff
@@ -0,0 +1,42 @@
+=========================
+Log Data Postprocessing
+=========================
+
+Log data includes a) data logged from the search results page rendered by
+Omega query template and, b) clicks recorded when a document link is
+clicked on the results page.
+
+Further processing of this raw data is handled by the ``postprocess.py``
+script. This script generates the final clickstream log file from the input
+search and click log files which can be used to train click models for clickstream
+data mining. It also creates a Query file that can be used by Xapian Letor module
+for generating its training files.
+
+The two functions defined in ``postprocess.py`` are:
+
+- ``postprocess.generate_final_log(search_log, clicks_log, final_log)``
```

Comment:
Perhaps `generate_combined_log()` would be better? Or `combine_logfiles()`, perhaps.